.eslintrc.js

```diff
@@ -21,6 +21,14 @@ module.exports = {
         "import/no-unresolved": "off",
         "import/prefer-default-export": "off",
       },
+      settings: {
+        'svelte3/ignore-styles': (attrs) => {
+          // to suppress 'css-unused-selector' warnings
+          if (attrs.global)
```

Comment:
Actually, I can't found documentation on the `<style global>` syntax we are using. Instead, I found `:global` selector in official documentation. We should switch to use it, then the warnings should go away.

Reply:
this feature is from svelte-preprocess https://github.com/sveltejs/svelte-preprocess#global-style

Reply:
We can try making it into a plain CSS file.

Reply:
@kiootic I moved the global styles into a separate CSS file. It still uses non-standard style tag attributes but the alternative is to use another rollup plugin to create a 2nd CSS bundle which I think is a waste.